mkosi/config.py

```diff
@@ -324,6 +326,8 @@ def to_efi(self) -> Optional[str]:
             Architecture.loongarch64 : "loongarch64",
         }.get(self)
 
+        # TODO should this have an explicit check+die?
```

Comment:
Why the comment here?

Reply:
All the other `to_*` methods have explicit checks if the architecture is supported and exit with a proper error message except for this one. That seemed like an oversight though on second look it seems like the `None` case is used somewhere and explicit checks are in some other places